controller/src/main/java/io/pravega/controller/task/Stream/TxnSweeper.java

```diff
@@ -168,27 +171,39 @@ public boolean isReady() {
         String scope = txn.getScope();
         String stream = txn.getStream();
         UUID txnId = txn.getTxnId();
-        log.debug("Host = {}, failing over committing transaction {}/{}/{}", failedHost, scope, stream, txnId);
-        return transactionMetadataTasks.writeCommitEvent(scope, stream, epoch, txnId, TxnStatus.COMMITTING)
+        // To persist the original requestId across hosts in TxnSweeper would require to add requestID in TxnResource.
```

Comment:
I'm not sure what this comment is trying to say. Is this something that is desirable but you are not addressing at the moment?

Reply:
Removed as all the tracing code related to transactions.